modules/dea/requirements.txt

```diff
@@ -0,0 +1,5 @@
+git+https://github.com/opendatacube/datacube-core.git@develop#egg=datacube
+git+https://github.com/GeoscienceAustralia/eo-datasets.git@develop#egg=eodatasets
+git+https://github.com/GeoscienceAustralia/digitalearthau.git@develop#egg=digitalearthau
```

Comment:
I presume we'll use stable branches for all of these once finalised? (or 'nci' branches if more control is needed...)

Reply:
Yep, as a first step `stable` branches.

I'm also thinking about having auto-deploys for stable version releases, which might involve having Travis build a conda or python package, upload to S3, and have `requirements.txt` point there instead of git repositories. Maybe even upload to PyPI automatically and use that. Rolling towards less manual steps.

We could also easily create a configuration to do `dea/develop` style module builds, that *do* point to the develop branches of everything, and is automatically kept up to date with them.